Watchman.Discord/Areas/Reactions/Services/ReactionsService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+using Watchman.Discord.Areas.Reactions.Contexts;
+
+namespace Watchman.Discord.Areas.Reactions.Services
+{
+    public static class ReactionsService
+    {
+        public static Action<ReactionContext> UserAddedReaction { get; set; }
+        public static Action<ReactionContext> UserRemovedReaction { get; set; }
+
+        public static Task OnReactionAddedEvent(SocketReaction reaction)
+        {
+            return Task.Run(() => UserAddedReaction(GetReactionContext(reaction)));
+        }
+
+        public static Task OnReactionRemovedEvent(SocketReaction reaction)
+        {
+            return Task.Run(() => UserRemovedReaction(GetReactionContext(reaction)));
+        }
```

Comment:
nie lepiej żeby to było w WorkflowBuilder

Reply:
w sumie można wykorzystać WorkflowBuilder ale nie za bardzo rozumiem do czego moglibyśmy to wykorzystać

ja zrobiłbym coś na wzór obecnego ControllersService 
tylko pewnie byłoby mniej rozbudowane
albo rozszerzyć obecne ControllersService o możliwość obsługi reakcji 

Możnaby zrobić tak
Że kontrolery mogłyby mieć metody, które przyjmowałyby jedynie ReactionContext
albo coś tego typu 

I te metody byłyby używane jako metody do obsługi reakcji 

Z tym że musiałby być jakiś system przydzielania odpowiedniej reakcji do odpowiedniej metody